autoscale/src/main/java/com/sequenceiq/periscope/monitor/handler/UpdateFailedHandler.java

```diff
@@ -27,16 +31,20 @@
     @Inject
     private ClusterService clusterService;
 
+    @Inject
+    private HistoryService historyService;
+
+    @Inject
+    private CloudbreakMessagesService messagesService;
+
     private final Map<Long, Integer> updateFailures = new ConcurrentHashMap<>();
 
     @Override
     public void onApplicationEvent(UpdateFailedEvent event) {
         long autoscaleClusterId = event.getClusterId();
         Cluster cluster = clusterService.findById(autoscaleClusterId);
-        if (cluster == null) {
-            return;
-        }
         MDCBuilder.buildMdcContext(cluster);
```

Comment:
This is a no-op block since findById throws NotFoundException if clusterId is not found.

Reply:
Ack. Curious what happens in case the Exception is thrown.

Reply:
Exceptions thrown out of onApplicationEvent are ignored.